coderedcms/models/wagtailsettings_models.py

```diff
@@ -378,3 +378,19 @@ class Meta:
         verbose_name=_('Google Maps API Key'),
         help_text=_('The API Key used for Google Maps.')
     )
+
+
+@register_setting(icon='fa-puzzle-piece')
+class MailchimpApiSettings(BaseSetting):
+    """
+    Settings for Mailchimp API services.
+    """
+    class Meta:
+        verbose_name = _('Mailchimp API Settings')
```

Comment:
Can remove the word "settings" here (and also on google api settings) since they are nested under the "settings" menu panel.